src/frontend/components/PublicVideoLiveJitsi/index.spec.tsx

```diff
@@ -30,23 +31,18 @@ jest.mock('../../data/appData', () => ({
     video: mockVideo,
   },
 }));
-
-const mockExecuteCommand = jest.fn();
-const mockJitsi = jest.fn().mockImplementation(() => ({
-  executeCommand: mockExecuteCommand,
-  addListener: jest.fn(),
+jest.mock('components/StudentLiveWrapper', () => ({
```

Comment:
What happen if you don't mock it? You mock a lot of child component in your tests, why are you applying this strategy ? I think we should not mock them if we can